sdk/nextjs/vercel.ts

```diff
@@ -0,0 +1,98 @@
+import { unstable_flag as flag } from '@vercel/flags/next'
+import { setupDevCycle } from './src/server/setupDevCycle'
+
+// mimic the Vercel JsonValue type since it is not exported
+type JsonValue =
```

Comment:
does the array support here get pulled out somewhere else?

Reply:
what do you mean?